pkg/kubelet/kubelet.go

```diff
@@ -1198,6 +1209,9 @@ type Kubelet struct {
 
 	// Manage user namespaces
 	usernsManager *usernsManager
+
+	// draManager handles Dynamic Resource Allocation requests.
+	draManager dra.Manager
```

Comment:
I'm not sure how I feel about putting this directly in the kubelet without plumbing it through the container manager (especially since the `dra` is a package under `cm`). Let me look through the rest of the code and get a feel for it.

Reply:
If we do this, I think we should move `dra` as a package directly under the kubelet instead of embedded under `cm`. And as part of that, I'm not actually sure how I feel about the name `dra` for this anymore. It's more of a `ResourceClaim` manager -- it's starting to feel like the package name should reflect that.

I was fine with `dra` under `cm` since it#s so deep and encapsulated -- but directly under `kubelet` it doesn't feel good as a name.

Reply:
OK, after thinking about this, I propose the following:

1. Continue to call the package `dra` but move it directly under the kubelet
2. Rename the API calls to `PrepareDynamicResources()`, `UnprepareDynamicResources()`, `GetDynamicResources()`

This will help distinguish it from the `Resources` managed by the `ContainerManager`